lib/adminware/schedule.rb

```diff
@@ -0,0 +1,82 @@
+require 'adminware'
+require 'adminware/config'
+require 'yaml'
+require 'fileutils'
+
+module Adminware
+  class Schedule
+    attr_accessor :name
+    attr_accessor :command
+
+    def initialize(host)
+      @host = host
+      @config = Adminware.config
+      @file = File.join(Adminware.root, 'var', "#{@host}_schedule.yaml")
+      @array = []
+
+      load_schedule
+    end
+
+    #Add a job to the schedule
+    def add_job
```

Comment:
This seems more suited to accepting the `command` and `name` values as arguments.  (Though it will mean refactoring the validation too.)